R/readGWLdata2.R

```diff
@@ -0,0 +1,121 @@
+# Copyright 2015 Province of British Columbia
```

Comment:
Hi Jay - Can you change this to `Copyright 2015 Jay Brown` (or whatever preferred name you go by) so that you retain copyright of the original work (but grant us license to use and  modify it - see [CONTRIBUTING](https://github.com/bcgov/bcgroundwater/blob/master/CONTRIBUTING.md)).